doc/man/8/ceph.rst

```diff
@@ -385,13 +430,22 @@ Usage::
 
 	ceph fs rm <fs_name> {--yes-i-really-mean-it}
 
-Subcommand ``authorize`` creates a new client that will be authorized for the
-given path in ``<fs_name>``. Pass ``/`` to authorize for the entire FS.
-``<perms>`` below can be ``r``, ``rw`` or ``rwp``.
+Subcommand ``rm_data_pool``  removes the specified pool from FS's list of
+data pools. File data on this pool will become unavailable. Default data pool
+cannot be removed.
 
 Usage::
 
-    ceph fs authorize <fs_name> client.<client_id> <path> <perms> [<path> <perms>...]
+    ceph fs rm_data_pool <fs-name> <pool name/id>
+
+Subcommand ``set`` sets or updates a FS setting value for given FS name.
+
+Usage::
+
+    ceph fs set <fs-name> <fs-setting> <value>
+
+.. TODO:: add ``fs feature ls`` and ``fs flag set <flag-name> <flag-val>
```

Comment:
@rishabh-d-dave There's no directive called "TODO" that Sphinx recognizes, so this line will cause the docs build to fail and will make this PR unmergeable. When you're ready, just remove this so that I can merge it.

Also, don't worry about getting all the commands into one PR. We can always make another PR.

Reply:
> @rishabh-d-dave There's no directive called "TODO" that Sphinx recognizes, so this line will cause the docs build to fail and will make this PR unmergeable. When you're ready, just remove this so that I can merge it.

Sure.
 
> Also, don't worry about getting all the commands into one PR. We can always make another PR.

True but these options should've been in man page long time ago but we (cephfs team) didn't do it for SO long. Hopefully I'll receive reply in 12 hours and then I'll remove this TODO and make this PR ready for merge.

Reply:
> @rishabh-d-dave There's no directive called "TODO" that Sphinx recognizes, so this line will cause the docs build to fail and will make this PR unmergeable. When you're ready, just remove this so that I can merge it.

I suggest https://pypi.org/project/restview/, you can compile and see see the changes you've made locally

Reply:
> > @rishabh-d-dave There's no directive called "TODO" that Sphinx recognizes, so this line will cause the docs build to fail and will make this PR unmergeable. When you're ready, just remove this so that I can merge it.
> 
> I suggest https://pypi.org/project/restview/, you can compile and see see the changes you've made locally

I've left TODO there deliberately.